webcompat/static/js/lib/issues.js

```diff
@@ -234,7 +234,7 @@ issues.ImageUploadView = Backbone.View.extend({
     var DELIMITER = '\n\n';
     var textarea = $('.js-Comment-text');
     var textareaVal = textarea.val();
-    var imageURL = _.template('![Screenshot of the site issue](<%= url %>)');
+    var imageURL = _.template('[![Screenshot of the site issue](<%= url %>)](<%= url %>)');
```

Comment:
> so that is one of my issue here. We have a function in bugform.js which is working already. Why do we need twice the code.

Can you expand on this? I don't quite follow. Or are you saying why don't we have a single method/module that is shared by bugform and issues?